scripts/xsslint_thresholds.json

```diff
@@ -9,7 +9,7 @@
         "javascript-jquery-insertion": 19,
         "javascript-jquery-prepend": 7,
         "mako-html-entities": 0,
-        "mako-invalid-html-filter": 9,
+        "mako-invalid-html-filter": 13,
```

Comment:
Why did this limit have to be increased?

Reply:
This was for the editor in the maintenance view, since we're showing the announcement unescaped for TinyMCE to edit. It looks like these have moved to the inline overrides though, so I've removed this limit increase.

Reply:
+1